### PR TITLE
fix: Reconcile gremlin description writes with the Databuilder

### PR DESCRIPTION
**Pull request by AndrewCiambrone**
Signed-off-by: Andrew Ciambrone <andrjc4@vt.edu>

### Summary of Changes

- Upgrades the metadata proxy requirements so that it uses amundsengremlin 0.0.9 which changes the description key to the same as the databuilders. 

- Removed the reset shards from the neptune proxy because that has been moved to amundsengremlin in 0.0.8

- fixed the default source to be "description" instead of "user" 

### Tests

_What tests did you add or modify and why? If no tests were added or modified, explain why. Remove this line_

### Documentation

_What documentation did you add or modify and why? Add any relevant links then remove this line_

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [X] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [X] PR includes a summary of changes.
- [x] PR adds unit tests, updates existing unit tests, **OR** documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes `make test`

### CODE REVIEW

File: metadata_service/config.py [hunk 1]
```diff
@@ -123,6 +123,7 @@ class NeptuneConfig(LocalGremlinConfig, LocalConfig):
     LOG_LEVEL = 'INFO'
 
     # PROXY_HOST FORMAT: wss://<NEPTUNE_URL>:<NEPTUNE_PORT>/gremlin
+    PROXY_HOST = os.environ.get('PROXY_HOST', 'localhost')
```
Comment (AndrewCiambrone) [inline metadata_service/config.py:126]: This was accidentally removed in a previous pr. Adding it back.

File: requirements.txt [hunk 2]
```diff
@@ -36,7 +36,7 @@ typing-extensions==3.7.4
 # A common package that holds the models deifnition and schemas that are used
 # accross different amundsen repositories.
 amundsen-common>=0.9.0
-amundsen-gremlin>=0.0.7
+amundsen-gremlin>=0.0.9
```
Comment (feng-tao) [inline requirements.txt:39]: could you update https://github.com/amundsen-io/amundsendatabuilder/blob/master/setup.py#L66 as well? the requirement.txt is only used in CI.

**Review by feng-tao — APPROVED** [review]

File: requirements.txt (hunk 2 above)
Comment (AndrewCiambrone) [reply requirements.txt:39]: Sure: https://github.com/amundsen-io/amundsendatabuilder/pull/473

Comment (AndrewCiambrone) [reply requirements.txt:39]: Opps or did you mean to move gremlin dependencies to be extras like: https://github.com/amundsen-io/amundsenmetadatalibrary/blob/master/setup.py#L26 ?

Comment (feng-tao) [reply requirements.txt:39]: nvm, I think you are correct, I somehow thought you are updating the databuilder.

**Pull request merged**